FONTLOG.txt

```diff
@@ -47,6 +47,14 @@ ChangeLog
 
 unreleased (GIT) <Libertinus> Version HEAD
 
+13 January 2021 (Felix Kopecky) <Libertinus> Version 7.032
```

Comment:
```suggest
```

The way I usually manage this change log is just to keep new bullet points for unreleased items under the unreleased heading at the top. Later when I actually cut a release I'll mark the date of the release and whatever the official release number ends up being. Any fonts built from non tagged releases (such as can be downloaded from this PR's check artifacts) will be automatically versioned with a number relative to the last tag anyway.